packages/donate-button-beta/src/components/widget/index.tsx

```diff
@@ -89,21 +94,29 @@ const nonProfitHeaderCss = cxs({
 		gridRow: '1 / 2'
 	}
 });
+
 const nonProfitInfoCss = cxs({
 	gridColumn: '2 / 3',
 	gridRow: '2 / 4'
 });
-const ctaCss = cxs({
+
+const cryptoContainerCss = cxs({
+	gridRow: '4 / 4',
```

Comment:
I didn't have the opportunity of look at the result but it seems that here should be `4 / 5`

Reply:
@martinbianchi fixed :) please check